config/locales/es.yml

```
@@ -242,31 +242,31 @@ es:
+      a:  "¡No! CodeBar se trata de conseguir que las personas se entusiasman con la programación. Los ayudaremos a aprender algunas habilidades básicas de codificación, así como cómo explorar y hacer las cosas por su cuenta. Para aquellos que quieren convertirse en desarrolladores, proporcionaremos ayuda y Orientación para ayudarles a lograr su objetivo."
     format:
       q: "¿Cuál es el formato de los talleres?"
-      a: "We try to have one coach for every two students. We will pair up students who want to work on the same tutorial, and assign them a coach. The rest just happens."
+      a:  "Tratamos de tener un entrenador para cada dos estudiantes. Vamos a emparejar a los estudiantes que quieran trabajar en el mismo tutorial y asignarles un entrenador. El resto solo sucede."
     laptop:
       q: "¿Necesito traer una computadora portátil a los talleres?"
-      a: "Yes, we suggest that you bring a laptop as it will be easier for you to carry on working on things after the workshops. It you don't have one, <a href=\"mailto:contact@codebar.io\">send us an email</a> so we can try and help out."
+      a: "Sí, le sugerimos que traiga una computadora portátil, ya que será más fácil para usted seguir trabajando en cosas después de los talleres. No tiene uno, <a href=\"mailto:contact@codebar.io\"> envíenos un correo electrónico </a? para que podamos intentar y ayudarlo."
     online_resources:
       q: "Hay tantos recursos en línea, ¿por qué debería ir a Codebar?"
-      a: "Coding can be intimidating and a lot of people find it hard to get started. By putting you in touch with people who can help and guide you we make the experience and learning process easier and more enjoyable."
+      a: "La codificación puede ser intimidante y muchas personas les resulta difícil comenzar. Al ponerte en contacto con personas que pueden ayudarlo y guiarlo, hacemos que la experiencia y el proceso de aprendizaje sean más fáciles y más agradables."
     preperations:
-      q: "I would like to coach at codebar. Is there anything I should do besides signing up?"
+      q:  "Me gustaría entrenar en CodeBar. ¿Hay algo que deba hacer además de inscribirse?"
       a_p1: "Si. Primero deberías leer nuestro"
       coaching_guide_tutorials: "tutoriales de guía de entrenador"
-      a_p2: ". Then spend sometime familiarising yourself with the"
-      a_p3: "<a href='http://tutorials.codebar.io'>tutorials</a>. The tutorials are open source so if there is something that you want to improve open an issue on the GitHub repository or send a pull request."
-      a_p4: "We don't expect the coaches to be familiar with all the technologies and languages we have material on."
+      a_p2:  ". Luego pasa alguna vez familiarizándote con el"
```

```suggestion
      a_p2: ". Luego pasa alguna vez familiarizándote con el"
```

And one here as well.